tests/sentry/backup/test_models.py

```diff
@@ -57,6 +59,36 @@ def import_export_then_validate(self):
         if res.findings:
             raise ValidationError(res)
 
+    def create_monitor(self):
```

Comment:
should we maybe move this to a separate file? group the files based on models? that way when new models are out and supported it's not all appended to this? kinda nit picky though

Reply:
Discussed offline; we decided to keep this in a single file for now for the benefit of possible future static completeness checks, but will re-asses splitting up as things grow.